src/python/views/demo_bio_01_funcoup/__init__.py

```diff
@@ -0,0 +1,307 @@
+import graphistry, os, pandas as pd, streamlit as st
+from components import GraphistrySt, URLParam
+from graphistry import PyGraphistry
+from css import all_css
+from time import sleep
+import logging
+
+############################################
+#
+#   DASHBOARD SETTINGS
+#
+############################################
+#  Controls how entrypoint.py picks it up
+
+
+app_id = 'app_00'
```

Comment:
can you change to a later demo ID so doesn't take precedence over the intro demo series?